app1React/src/singleSpaEntry.js

```diff
@@ -15,10 +15,7 @@ export function bootstrap(props) {
 }
 
 export function mount(props) {
-	return reactLifecycles.mount(props).then((rootComponent) => {
-        rootComponent.setStore(props.customProps.store);
```

Comment:
See https://github.com/CanopyTax/single-spa-react/issues/36. There was no need to call a setStore or setGlobalEventDistributor function at all, since they were already being passed as props through to the react component.